### ISSUE TITLE
channel from self node has no policy when updating the channel fees

### ISSUE THREAD

**Issue by jaccos** [lightningnetwork/lnd#5861]
### Background

when trying to make the base fee from a channel  to zero i get this message -> channel from self node has no policy 

### Your environment
umbrel 0.4.4

### Steps to reproduce

update channel

### Expected behaviour

update the channel

### Actual behaviour

nothing

**Comment by Roasbeef** [comment]
Looks like your node hasn't advertised the outgoing edge for a given channel. How old is the channel in question (can check in the logs or based on your args to the command)? Do you know the short channel ID of the channel?

**Comment by jaccos** [comment]
All the (open ) channels have the same issue.
What is the outgoing edge and where to find?
Do not no how to find in the command. Do you mean lncli?

**Comment by ohenrik** [comment]
I think have this exact problem. I had a channel open in the mempool due to the increase in on-chain fees. I think the transaction was deleted from the mempool. So in order to get the fee bumped and channel opened I had to re-broadcast the transaction and bump the fee using CPFP. 

Now that the channel is open, it can't be found in the channel graph, I can't update the channel policy or close the channel. The command just hangs when I try to close the channel. 

```
lncli getchaninfo 86...36 (<-- the lnd short channel id)
[lncli] rpc error: code = Unknown desc = edge not found
```
I suspect the peer does not know it was finally opened.

Is there any way to recover from this by closing the channel?

**Comment by guggero** [comment]
@ohenrik no, if the 2 week period for a channel confirmation is over, the peer will "forget" about the channel and you need to force close. See https://github.com/lightningnetwork/lnd/issues/7434 and https://github.com/lightningnetwork/lnd/issues/7481 for possible future mitigations.
